### DISCUSSION TITLE
hyprdots update and vscode profile lost

### DISCUSSION THREAD

**Discussion by Gedankenn** [prasanthrangan/hyprdots#164] [Q&A]
Everytime i update the hyprdots its like a fresh install of vscode, i lost all my plugins and saved configurations for vscode, you guys know how to workaround it?, thanks for the help

**Comment by prasanthrangan** [comment]
oh sorry about that, the `.config/Code/User` directory is moved to backup directory everytime you update... so please copy it back from backup to .config dir...

I'll fix this by changing
```
${HOME}/.config/Code|User|visual-studio-code-bin
```
to
```
`${HOME}/.config/Code/User|settings.json|visual-studio-code-bin`
```

so that only the settings.json setting is updated, and user directory is preserved!

**Reply by Gedankenn** [reply]
Oh, thanks, i really appreciate the help. best wishes. :smile:

**Reply by prasanthrangan** [reply]
fixed in 79807bd
